util/registry/queuefile.py

```diff
@@ -1,9 +1,13 @@
+from multiprocessing.queues import Empty, Queue
+
+
 class QueueFile(object):
     """ Class which implements a file-like interface and reads QueueResult's from a blocking
       multiprocessing queue.
   """
 
-    def __init__(self, queue, name=None):
+    def __init__(self, queue, name=None, timeout=None):
+        assert isinstance(queue, Queue)
```

Comment:
This is breaking unit tests since it's using a FakeQueue.

Reply:
Removed